vaadin-form-layout-flow-parent/vaadin-form-layout-flow/src/main/java/com/vaadin/flow/component/formlayout/FormLayout.java

```diff
@@ -449,4 +449,16 @@ public FormItem addFormItem(Component field, Component label) {
         add(formItem);
         return formItem;
     }
+
+    /**
+     * Set the width of the label when aside.
```

Comment:
```suggestion
     * Set the width of side-positioned labels.
```
Might be a bit clearer?